lambda_functions/app.py

```diff
@@ -351,4 +404,32 @@ def item_submission(event, context):
         return {
             "statusCode": 400,
             "body": "Could not create item and/or submission. Check HTTP POST payload. Exception: {}".format(e)
-        }
\ No newline at end of file
+        }
+
+
+def get_open_item_for_user(event, context):
+
+    try:
+        # get user id (str) from path
+        id = event['pathParameters']['id']
```

Comment:
Ideally, we will read the Cognito userId out of the context object. 
However, I think we should change this later, after we implemented the login functionality.

Reply:
I see. Yeah, it's a little difficult to debug without the login, maybe we can do that next week.